src/AssemblerHttpServer-ReadGraph.cpp

```diff
@@ -31,18 +31,53 @@ void Assembler::exploreReadGraph(
 }
 
 
+bool parseCommaSeparatedReadIDs(string& commaSeparatedReadIDs, vector<OrientedReadId>& readIds, ostream& html){
+    readIds.clear();
+    string token;
+
+    for (auto& c: commaSeparatedReadIDs){
+        if (c == ','){
+            try {
+                OrientedReadId readID(token);
+                readIds.emplace_back(readID);
```

Comment:
I would not bother with `emplace_back` here and use the more standard `push_back` instead. Unless I misunderstand something, it provides no benefits for this code where you are just dealing with integers.

Reply:
As written, I dont think it would provide any benefit. If instead I did:
`readIds.emplace_back(token);`

It would avoid copying or moving the `readId` object. Would you be ok with this implementation?

Reply:
No, the compiler would optimize all of that away anyway. I think `emplace_back` is meant for situations where there is a significant cost in object construction, and/or object construction is complex enough that the compiler cannot do a good job inlining it.

And more in general, in areas of the code that are not important for performance, we should avoid optimizations that make the code less readable.